### PR TITLE
Add Darwinia Crab

### PR DESCRIPTION
**Pull request by AurevoirXavier**
Due to the marketing concerns. We decide to rename the **Crab** to **Darwinia Crab** at current stage. Thanks!

https://github.com/darwinia-network/darwinia/issues/500

### CODE REVIEW

File: frontend/src/components/List/Column.tsx [hunk 1]
```diff
@@ -73,6 +73,7 @@ const ICONS = {
   'ladder-node': ladderIcon,
   'cennznet-node': cennznetIcon,
   Crab: crabIcon,
+  "Darwinia Crab": crabIcon,
```
Comment (dvdplm) [inline frontend/src/components/List/Column.tsx:76]:
```suggestion
  'Darwinia Crab': crabIcon,
```

**Review by dvdplm — COMMENTED** [review]
LGTM, but shouldn't we remove the `Crab` member?

File: frontend/src/components/List/Column.tsx (hunk 1 above)
Comment (AurevoirXavier) [reply frontend/src/components/List/Column.tsx]: Done.

**Comment by AurevoirXavier** [comment]
> LGTM, but shouldn't we remove the `Crab` member?

It might be used later. But It's OK. Let me remove it.

**Comment by dvdplm** [comment]
CI failure looks legit – @AurevoirXavier  can you check that the tests pass on your end please?

**Comment by AurevoirXavier** [comment]
> CI failure looks legit – @AurevoirXavier can you check that the tests pass on your end please?

Fixed.

File: frontend/src/components/List/Column.tsx [hunk 2]
```diff
@@ -106,8 +105,8 @@ export namespace Column {
           </span>
         </Tooltip>
       ) : (
-        '-'
-      );
+          '-'
```
Comment (dvdplm) [inline frontend/src/components/List/Column.tsx]: funny whitespace here

File: frontend/src/components/List/Column.tsx [hunk 3]
```diff
@@ -360,8 +359,8 @@ function formatStamp(stamp: Types.Timestamp): string {
   return hours
     ? `${hours}h ago`
     : minutes
-    ? `${minutes}m ago`
-    : `${seconds}s ago`;
+      ? `${minutes}m ago`
+      : `${seconds}s ago`;
```
Comment (dvdplm) [inline frontend/src/components/List/Column.tsx]: …and here.

**Review by dvdplm — CHANGES_REQUESTED** [review]
pls fix whitespace

**Review by dvdplm — APPROVED** [review]

**Comment by dvdplm** [comment]
@maciejhirsz ok to merge?

**Comment by AurevoirXavier** [comment]
> pls fix whitespace

Already fixed.

See [revert: auto format](https://github.com/paritytech/substrate-telemetry/pull/284/commits/ab348e15d347451bf173d00273aa077ce8d5e203).

**Review by maciejhirsz — APPROVED** [review]

**Comment by maciejhirsz** [comment]
LGTM!

**Pull request merged**